packages/material-ui-lab/src/TreeView/useTreeState.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import TreeViewContext from './TreeViewContext';
+
+function useTreeState() {
```

Comment:
Do we need a module for it? Can consumers use `React.useContext(TreeViewContext)` directly? It's what we do with the Table.

Reply:
> Do we need a module for it?

Yes

> Can consumers use React.useContext(TreeViewContext) directly? It's what we do with the Table.

That's not a good API. You want to hide as much implementation detail as possible.

Reply:
It's a private module, right?

Reply:
I like having a separate module for it just for separation and syntax sake but I'm fine with either in the long term.